test/e2e/e2e_test.go

```diff
@@ -104,6 +104,26 @@ var _ = Describe("e2e", func() {
 			Expect(err).To(BeNil())
 			Expect(stdout).To(ContainSubstring("Welcome to nginx!"))
 		})
+		It("Deploy then remove valid edgeworkload to registered device", func() {
+			// given
+			err := device.Register("dnf install ansible -y")
```

Comment:
I think you can remove here this `dnf` command

Reply:
done